Nos/Service/Analytics.swift

```diff
@@ -297,4 +297,8 @@ class Analytics {
     func cancelledUploadSourceSelection() {
         track("Cancelled Upload Source Selection")
     }
+
+    func openedMentions() {
+        track("Mentions Opened")
```

Comment:
How about "Mentions Autocomplete Opened"? I think it's a little more clear.

Reply:
I wanted to use something anyone could easily understand on the dashboard, but  "Mentions Autocomplete Opened" definitely looks clearer.